variable/emissions/emissions.yaml

```diff
@@ -190,39 +190,67 @@ Carbon Sequestration|Land Use|Soil Carbon Management:
                 techniques
    unit: Mt CO2/yr
 
-Emissions|BC:
+# Variables related to the CO2 emission by power plant
+Emissions|CO2|Electricity|Biomass|Traditional:
+   description: CO2 emmissions of a typical Traditionnal Biomass power plant
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Biomass|New and imported :
+   description: CO2 emmissions of a typical new biomass power plant
+                biomass from dedicated energy crops and biomass imports.
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Coal:
+   description: CO2 emmissions of a typical coal power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Gas|OCGT:
+   description: CO2 emmissions of a typical Open Cycle Gas Turbine power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Lignite:
+   description: CO2 emmissions of a typical Lignite power plant
+   unit: tons/MWh
+
+Emmissions|CO2|Electricity|Nuclear:
+   description: CO2 emmissions of a typical nuclear power plant
+   unit: tons/MWh
+
+Emissions|Black Carbon:
    description: Total black carbon emissions
    unit: Mt BC/yr
 
-Emissions|BC|AFOLU:
+Emissions|Black Carbon|AFOLU:
    description: Black carbon emissions from agriculture, forestry and other
                 land use (IPCC category 3)
    unit: Mt BC/yr
 
-Emissions|BC|Energy:
+Emissions|Black Carbon|Energy:
    description: Black carbon emissions from energy use on supply and demand
                 side, including fugitive emissions from fuels (IPCC category
                 1A, 1B)
    unit: Mt BC/yr
 
-Emissions|BC|Energy|Demand|Industry:
+Emissions|Black Carbon|Energy|Demand|Industry:
    description: Black carbon emissions from fuel combustion in industry (IPCC
                 category 1A2)
    unit: Mt BC/yr
 
-Emissions|BC|Energy|Demand|Residential and Commercial:
+Emissions|Black Carbon|Energy|Demand|Residential and Commercial:
    description: Black carbon emissions from fuel combustion in residential,
```

Comment:
does this mean that is emissions coming from residentials burning something at home? Or emissions related to the final energy that they consume?